LeadAlerts.Web/Domain/Configuration.cs

```diff
@@ -6,12 +6,14 @@ public class Credentials
     {
         public static string AccountSID
         {
-            get { return WebConfigurationManager.AppSettings["TwilioAccountSid"]; }
+            get { return WebConfigurationManager.AppSettings["TwilioAccountSid"] ??
+                    "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; }
```

Comment:
Why did you add a fallback value? With this value the application will return a resource not found (404), so I can't see a benefit on this approach. Is there something I'm missing here?